src/index.ts

```diff
@@ -1 +1,48 @@
-console.log("hello, world!");
+const fs = require('fs');
+import chalk from "chalk";
+import { drawBoard } from './board/board';
+
+let data = [
```

Comment:
If the data doesn't change, replace `let` with `const`

Reply:
Done